enterprise/server/cmd/ci_runner/main.go

```diff
@@ -1986,6 +1986,7 @@ func (ws *workspace) config(ctx context.Context) error {
 		{"user.email", "ci-runner@buildbuddy.io"},
 		{"user.name", "BuildBuddy"},
 		{"advice.detachedHead", "false"},
+		[]string{"credential.interactive", "false"},
```

Comment:
nit: match syntax of other entries

```suggestion
		{"credential.interactive", "false"},
```